js/button/shared/Button.svelte

```diff
@@ -190,8 +190,14 @@
 	}
 
 	.huggingface {
-		border: var(--button-border-width) solid
-			var(--button-secondary-border-color);
-		background: var(--background-fill-primary);
+		background: black;
```

Comment:
The "huggingface" variant should _not_ change depending on the theme, in order to preserve the Hugging Face branding